packages/slate/src/changes/with-schema.js

```diff
@@ -67,42 +66,26 @@ function normalizeNodeAndChildren(change, node, schema) {
     normalizeNode(change, node, schema)
     return
   }
+  const checkedKeys = []
```

Comment:
Can you change this variable to `keys`?

Reply:
Hi, is there any special reasons about changing the var name?  Personally I would prefer a bit adjective on the var name to help me remember what I have done when I read the code again.

Reply:
Fair enough, in that case can you call it `normalizedKeys` instead?

Reply:
Fixed by directly update on the last commit~